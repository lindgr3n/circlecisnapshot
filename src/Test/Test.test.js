import React from 'react'
import ReactDOM from 'react-dom'
import Test from './Test'
import {shallow} from 'enzyme'

describe('<Test>', () => {
    it('Should render', () => {
        const root = document.createElement('div')
        ReactDOM.render(<Test />, root)
    })

    it('Should render with snapshot', () => {
        const output = shallow(<Test />)
        expect(output).toMatchSnapshot()
    })
})