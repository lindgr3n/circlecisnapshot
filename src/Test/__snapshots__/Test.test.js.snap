// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test> Should render with snapshot 1`] = `
<div>
  Test
  <Second />
</div>
`;
